osgp/shared/osgp-ws-smartmetering/pom.xml

```diff
@@ -43,6 +43,24 @@
               <clearOutputDir>false</clearOutputDir>
             </configuration>
           </execution>
+          <execution>
+            <id>smart-metering-external-interface-integration</id>
+            <goals>
+              <goal>xjc</goal>
+            </goals>
+            <configuration>
+              <bindingDirectory>src/main/resources/schemas</bindingDirectory>
+              <bindingFiles>sm-external-interface.xjb</bindingFiles>
+              <schemaDirectory>src/main/resources/schemas</schemaDirectory>
+              <xmlschema>true</xmlschema>
+              <schemaFiles>sm-external-interface.xsd</schemaFiles>
+              <!--              <schemaFiles>sm-external-interface.xsd</schemaFiles>-->
+              <staleFile>${project.build.directory}/jaxb2/.schema-smart-metering-external-stale-flag</staleFile>
+              <!-- USE THIS ONLY FOR THE FIRST EXECUTION -->
+              <clearOutputDir>false</clearOutputDir>
```

Comment:
I think clearOutputDir can be removed. The default is true which clears the output directory before generating files. I think that's always ok.

Reply:
ik heb het uitgeprobeerd, dan blijven er maar twee packages over... dus ik laat het even zo.